packages/lib/testing/test-utils.ts

```diff
@@ -379,6 +380,7 @@ async function setupDatabase(id: number = null, options: any = null) {
 	await clearSettingFile(id);
 	await loadKeychainServiceAndSettings(options.keychainEnabled ? KeychainServiceDriver : KeychainServiceDriverDummy);
 
+	reg.setDb(databases_[id]);
```

Comment:
Without this change, `reg.scheduleSync` logs a warning related to a missing database during tests.